src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -449,6 +454,11 @@ export default {
 			const heroArea = contentGroups.campaignHeroCg;
 			return heroArea;
 		},
+		partnerAreaContent() {
+			const contentGroups = this.pageData?.page?.contentGroups;
+			const partnerArea = contentGroups?.partnerAreaCg;
```

Comment:
I see now, I need to refactor the contentfulUtil that names these contentGroups... will look at that next week.

Reply:
I think this is all good. "heroLogo" was just a leftover from a copy/paste.